google-cloud-firestore/src/test/java/com/google/cloud/firestore/LocalFirestoreHelper.java

```diff
@@ -191,6 +198,31 @@ public void setShortValue(@Nullable Short shortValue) {
     }
   }
 
+  public static class FooModel {
```

Comment:
Do you mind re-using the SingleField example Pojo that exists in this class?

Reply:
Done